misc/docusaurus/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Why is this required?

Reply:
This is added by the newest version of yarn, ie 4.0.2. Not sure if we should keep using it, or switch to something else.

Reply:
@alexiscolin, I'm calling you as the JavaScripter-in-chief, keep this file or burn it with fire? :fire:

Reply:
This line is important for yarn v4 to create  `node_modules` building strategy. `pnp` (another strategy - used by default now) will create plug and play system that doesn't rely on usual modules folder. That is faster but less compliant with the whole ecosystem (that can be discussed). 

This line has no impact on yarn v<4, so maybe we can keep it for now and for people who uses the v4 but propose a full change in another PR as this question is not related with as this one: DX vs analytics.